notebooks/README.md

```diff
@@ -48,9 +49,7 @@ and the NVIDIA P100 provided in the free tier, we will fine tune a LoRA.
 1. You'll need a Kaggle account, which you can create by visiting [Kaggle's Sign-up Page](https://www.kaggle.com/account/login?phase=startRegisterTab&returnUrl=%2F).
 1. To use Kaggle's accelerators, you'll have to verify your account with a phone number. Visit the [account settings page](https://www.kaggle.com/settings) and select "Phone Verification".
 
-
-**NOTE: At present, you'll need to download the notebook and upload it to Kaggle. The following steps will walk you through uploading the notebook. Once this repository is open sourced, we will make an 'Open in Colab' button**
-
+##### NOTE: At present, you'll need to download the notebook and upload it to Kaggle. The following steps will walk you through uploading the notebook. Once this repository is open sourced, we will make an 'Open in Colab' button**
```

Comment:
It can be another PR, but this formatting is nicer:

https://github.com/orgs/community/discussions/16925

Reply:
simple enough to change, will do that as part of this PR